test/memory.cpp

```diff
@@ -773,7 +769,7 @@ af_err alloc_fn(af_memory_manager manager, void **ptr,
         get_memory_pressure_fn(manager, &pressure);
         float threshold;
         af_memory_manager_get_memory_pressure_threshold(manager, &threshold);
-        if (pressure > threshold) { signal_memory_cleanup_fn(manager); }
+        if (pressure >= threshold) { signal_memory_cleanup_fn(manager); }
 
```

Comment:
The changes in this file basically reflect the change in #2801 Is that correct ?